Ensure MacPorts libiconv and icu are found even if -I/opt/local/include
is not in CPPFLAGS.
--- include/libxml/encoding.h.orig	2022-05-02 07:10:21.000000000 -0500
+++ include/libxml/encoding.h	2022-09-19 22:55:35.000000000 -0500
@@ -25,7 +25,7 @@
 #include <libxml/xmlversion.h>
 
 #ifdef LIBXML_ICONV_ENABLED
-#include <iconv.h>
+#include <@ICONV_PREFIX@/include/iconv.h>
 #endif
 
 #ifdef __cplusplus
